zipkin-lens/src/types/Dependencies.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+type Dependencies = {
```

Comment:
Similarly for everything in this folder, I'd define this where it's used. Being at this top level `src/types` location gives too little context on what this type is.

Reply:
I realized we could think about this in another PR. I think we already have this folder hierarchy

```
actions
  dependencies-action.js
  trace-action.js
components
  DependenciesPage
  TracePage
reducers
  dependencies
  trace
```

I would strongly suggest that at some point we move things to something like

```
components
  DependenciesPage
    index
    actions
    reducer
  TracePage
    index
    actions
    reducer
```

Action / reducer / component (and now types) are highly coupled so maintenance is simpler by keeping them together rather than separate.